webpack/assets/javascripts/react_app/components/SearchBar/__tests__/integration.test.js

```diff
@@ -2,85 +2,26 @@ import React from 'react';
 import API from '../../../API';
 import IntegrationTestHelper from '../../../common/IntegrationTestHelper';
 import { SearchBarProps } from '../SearchBar.fixtures';
-import SearchBar, { reducers } from '../index';
+import SearchBar from '../index';
+import { reducers } from '../../AutoComplete';
 import bookmarksReducer from '../../../redux/reducers/bookmarks';
 
 jest.mock('../../../API');
 jest.mock('lodash/debounce', () => jest.fn(fn => fn));
-jest.mock('uuid/v1', () => jest.fn(fn => '1234'));
 global.Turbolinks = {
   visit: jest.fn(),
 };
 
 const combinedReducers = { ...reducers, bookmarks: bookmarksReducer };
 
-describe('SearchBar integration test', () => {
+describe('Autocomplete integration test', () => {
```

Comment:
There is already `Autocomplete integration test`. Shouldn't it be `SearchBar integration test`?